### ISSUE TITLE
Add Opal v0.7.0 and v0.7.1

### ISSUE THREAD

**Issue by ggrossetie** [opal/opal-cdn#2]

**Comment by ggrossetie** [comment]
I tried to create the dist for the v0.7.0 branch but I get the following error:

```
$ rake dist
WARNING: Opal::Sprockets::Environment is deprecated. Please use Opal::Server directly or append Opal.paths to the environment manually.
* building opal...             done. (development: 350.01KB, minified: 222.78KB, gzipped:  39.24KB).
* building date...             done. (development:   7.26KB, minified:   4.88KB, gzipped:   1.19KB).
* building buffer...           done. (development:  36.46KB, minified:  25.87KB, gzipped:   4.69KB).
* building stringio...         done. (development:   7.62KB, minified:   5.47KB, gzipped:   1.40KB).
* building json...             done. (development:   7.91KB, minified:   5.52KB, gzipped:   1.44KB).
* building singleton...        done. (development:   2.96KB, minified:   2.27KB, gzipped:   0.70KB).
* building set...              done. (development:   9.18KB, minified:   6.77KB, gzipped:   1.79KB).
* building forwardable...      done. (development:   6.03KB, minified:   4.73KB, gzipped:   0.94KB).
* building math...             done. (development:  10.94KB, minified:   7.62KB, gzipped:   1.26KB).
* building time...             done. (development:   0.73KB, minified:   0.53KB, gzipped:   0.30KB).
* building benchmark...        done. (development:   0.82KB, minified:   0.61KB, gzipped:   0.33KB).
* building opal-parser...      done. (development: 578.88KB, minified: 452.94KB, gzipped:  73.98KB).
* building process...          done. (development:   0.88KB, minified:   0.64KB, gzipped:   0.27KB).
* building strscan...          done. (development:   4.74KB, minified:   3.19KB, gzipped:   0.91KB).
* building rbconfig...         done. (development:   1.17KB, minified:   0.95KB, gzipped:   0.45KB).
* building iconv...            done. (development:   0.25KB, minified:   0.16KB, gzipped:   0.13KB).
* building pathname...         done. (development:   2.56KB, minified:   1.86KB, gzipped:   0.67KB).
* building promise...          done. (development:  14.97KB, minified:  10.89KB, gzipped:   2.37KB).
* building dir...              done. (development:   0.36KB, minified:   0.27KB, gzipped:   0.20KB).
* building enumerator...       done. (development:   0.26KB, minified:   0.16KB, gzipped:   0.13KB).
* building delegate...         done. (development:   2.02KB, minified:   1.55KB, gzipped:   0.61KB).
* building native...           done. (development:  23.96KB, minified:  17.20KB, gzipped:   3.61KB).
* building encoding...         done. (development:  49.93KB, minified:  33.19KB, gzipped:   7.05KB).
* building file...             done. (development:   0.36KB, minified:   0.27KB, gzipped:   0.20KB).
* building fileutils...        done. (development:   0.26KB, minified:   0.16KB, gzipped:   0.13KB).
* building template...         done. (development:   2.60KB, minified:   1.88KB, gzipped:   0.68KB).
* building erb...              done. (development:   3.70KB, minified:   2.74KB, gzipped:   0.89KB).
* building sourcemap...        done. (development:  30.42KB, minified:  22.46KB, gzipped:   4.83KB).
* building yaml...             done. (development:  25.26KB, minified:  18.21KB, gzipped:   3.84KB).
* building ostruct...          done. (development:   4.11KB, minified:   3.02KB, gzipped:   0.94KB).
* building pp...               done. (development:   1.22KB, minified:   0.87KB, gzipped:   0.44KB).
* building nodejs...           done. (development:   9.23KB, minified:   6.86KB, gzipped:   1.80KB).
* building observer...         done. (development:   4.11KB, minified:   3.00KB, gzipped:   0.74KB).
* building base64...           done. (development:   4.21KB, minified:   2.63KB, gzipped:   0.86KB).
* building opal-source-maps... done. (development:  34.13KB, minified:  25.26KB, gzipped:   5.34KB).
* building securerandom...     done. (development:   0.86KB, minified:   0.60KB, gzipped:   0.35KB).
* building phantomjs...        done. (development:   0.89KB, minified:   0.66KB, gzipped:   0.34KB).
* building thread...           done. (development:   6.57KB, minified:   4.76KB, gzipped:   1.23KB).
* building source_map...       done. (development:  30.12KB, minified:  22.23KB, gzipped:   4.81KB).
* building minitest...         rake aborted!
Opal::Builder::MissingRequire: A file required by "minitest" wasn't found.
can't find file: "optparse" in ["/workspace/opal/opal", "/workspace/opal/stdlib", "/workspace/opal/lib"]
  (in /workspace/opal/stdlib/minitest.rb)
/workspace/opal/lib/opal/builder.rb:45:in `rescue in build_str'
/workspace/opal/lib/opal/builder.rb:38:in `build_str'
/workspace/opal/lib/opal/sprockets/processor.rb:75:in `evaluate'
/var/lib/gems/1.9.1/gems/tilt-1.4.1/lib/tilt/template.rb:103:in `render'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/context.rb:197:in `block in evaluate'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/context.rb:194:in `each'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/context.rb:194:in `evaluate'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/processed_asset.rb:12:in `initialize'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:374:in `new'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:374:in `block in build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:395:in `circular_call_protection'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:373:in `build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:94:in `block in build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/caching.rb:51:in `cache_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:93:in `build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:287:in `find_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:61:in `find_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/bundled_asset.rb:16:in `initialize'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:377:in `new'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:377:in `build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:94:in `block in build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/caching.rb:51:in `cache_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:93:in `build_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:287:in `find_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/index.rb:61:in `find_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/environment.rb:75:in `find_asset'
/var/lib/gems/1.9.1/gems/sprockets-2.12.3/lib/sprockets/base.rb:295:in `[]'
/workspace/opal/tasks/building.rake:33:in `block (2 levels) in <top (required)>'
/workspace/opal/tasks/building.rake:29:in `each'
/workspace/opal/tasks/building.rake:29:in `block in <top (required)>'
Tasks: TOP => dist
(See full trace by running task with --trace)
```

Same error with tag `v0.7.1`.

```
# find . -name optparse
./test/cruby/lib/optparse
./test/cruby/test/optparse
./test/cruby/sample/optparse
# bundle -v    
Bundler version 1.8.5
# ruby -v 
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
```

**Comment by elia** [comment]
The problem is that I erroneously included minitest in the stdlib misinterpreting what CRuby does and also it requires a number of files to be stubbed beforeaking it run properly. 

Long story short it should be skipped in the dist task for all 0.7 releases.

**Comment by ggrossetie** [comment]
Resolved by #3 and #4

**Comment by elia** [comment]
👍🏼
